src/Workspaces/CSharpTest/Formatting/EditorConfigOptionParserTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Formatting;
+using Xunit;
+using static Microsoft.CodeAnalysis.CSharp.Formatting.CSharpFormattingOptions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Formatting
+{
+    public class EditorConfigOptionParserTests
+    {
+        [Theory]
+        [InlineData("expressions", SpacingWithinParenthesesOption.Expressions)]
+        [InlineData("type_casts", SpacingWithinParenthesesOption.TypeCasts)]
+        [InlineData("control_flow_statements", SpacingWithinParenthesesOption.ControlFlowStatements)]
+        [InlineData("expressions, type_casts", SpacingWithinParenthesesOption.Expressions)]
+        [InlineData("type_casts, expressions, , ", SpacingWithinParenthesesOption.TypeCasts)]
+        [InlineData("expressions ,  ,  , control_flow_statements", SpacingWithinParenthesesOption.ControlFlowStatements)]
+        [InlineData("expressions ,  ,  , control_flow_statements", SpacingWithinParenthesesOption.Expressions)]
+        [InlineData(",  ,  , control_flow_statements", SpacingWithinParenthesesOption.ControlFlowStatements)]
```

Comment:
Alternatively, get rid of a bunch of brackets and just put `,` between the atrributes